packages/components/pill/README.mdx

```diff
@@ -8,58 +8,41 @@ storybook: 'https://v4-f36-storybook.netlify.app/?path=/story/components-pill--b
 typescript: ./src/Pill.tsx
 ---
 
-import { InfoCircleIcon } from '@contentful/f36-icons';
-
 Pills represent a set of one or multiple objects which can be entered or changed by the user. By default, pills can be removed by the user. When necessary, the order of pills can be changed by the user using drag and drop. They are used to represent complex entities in small blocks such as contact details. Contentful Pills are are used to render transition locales and tag elements.
 
-## How to use Pill
+## Import
 
-- use Pill to represent data like contact details or tagging
-- it can be used for web content that needs to be labeled, categorized, or organized using keywords that describe them.
+```js static=true
+import { Pill } from '@contentful/f36-components';
+// or
+import { Pill } from '@contentful/f36-pill';
+```
 
-## Code examples
+## Examples
 
-Example of usage with draggable and close option
+### Using with draggable and close option
 
-```jsx
-// import { Pill } from '@contentful/f36-pill';
+```jsx file=./examples/DraggableAndClosePillExample.tsx
```

Comment:
Don't know what is a better way to improve those two examples, but they feel kind of useless. Especially the second one with custom dragHandleComponent. WDYT, can we improve them somehow?

Reply:
I agree with you, it seems that the second one could be removed

Maybe there are some use cases in webapp

Reply:
I decided to keep those basic examples, just in case someone would need to just copy a basic one, and I added an example with react-sortable-hoc. let me know what do you think :)